dunecore/Geometry/gdml/dunevd10kt_3view_30deg_v5_refactored_1x8x14ref.gdml

```diff
@@ -224,6 +224,11 @@
    <fraction n="0.53" ref="fibrous_glass"/>
   </material>
 
+  <material name="FR4SussexAPA">
+   <D value="1.75" unit="g/cm3"/>
+   <fraction n="1" ref="FR4"/>
+  </material>
+
```

Comment:
Is this change expected?

Reply:
I believe this is harmless. It is likely the result of the file gdmlMaterials.pm not being versioned. That material was included for HD after VD v5 was originally created. That file gets included in any generated gdml. So when regenerating the v54 gdmls they newly pick up these changes to the materials definitions.

Reply:
Thanks for confirming, I figured it would be benign as the material was not used anywhere.